ocaml/libs/tracing/tracing.ml

```diff
@@ -51,7 +51,14 @@ module Status = struct
   type status_code = Unset | Ok | Error [@@deriving rpcty]
 
   type t = {status_code: status_code; description: string option}
-  [@@deriving rpcty]
+end
+
+module Attributes = Map.Make (String)
```

Comment:
Is this possible? We should keep functions related to attributes in a module.

```
module Attributes = struct
  open Map.Make (String)
  let of_list list = List.to_seq list |> of_seq
end
```

Reply:
I wasn't aware this was possible, this looks a lot neater